charts/argo-templates/scripts/create-bq-dataset.sh

```diff
@@ -0,0 +1,13 @@
+set -euo pipefail
+
+# Create a dataset, default expiration of 7 days
```

Comment:
relic of a comment from when I had the expiration hard coded; can remove the "default expiration of 7 days"